dla_cnn/spectra_utils.py

```diff
@@ -12,7 +12,7 @@ def get_lam_data(loglam, z_qso, REST_RANGE):
     loglam: np.ndarray
     z_qso: float
     REST_RANGE: list
-        Lowest rest wavelength to search, highest rest wavelength,  number of pixels in the search
+        Lowest rest wavelength to search, highest rest wavelength
```

Comment:
If you make this change, will the original code be broken?
Let's avoid that.

Reply:
It won't affect too much module ,maybe REST_RANGE[2] can be kept but not be used,I have updated the code